packages/shorebird_cli/lib/src/android_sdk.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:io';
+
+import 'package:collection/collection.dart';
+import 'package:path/path.dart' as p;
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/platform.dart';
+
+// https://developer.android.com/studio/command-line/variables.html#envar
+const kAndroidHome = 'ANDROID_HOME';
+const kAndroidSdkRoot = 'ANDROID_SDK_ROOT';
+
+/// A reference to a [AndroidSdk] instance.
+final androidSdkRef = create(AndroidSdk.new);
+
+/// The [AndroidSdk] instance available in the current zone.
+AndroidSdk get androidSdk => read(androidSdkRef);
+
+class AndroidSdk {
+  String? get path {
+    String? androidHomeDir;
+    if (platform.environment.containsKey(kAndroidHome)) {
+      androidHomeDir = platform.environment[kAndroidHome];
+    } else if (platform.environment.containsKey(kAndroidSdkRoot)) {
+      androidHomeDir = platform.environment[kAndroidSdkRoot];
+    } else if (platform.isLinux) {
+      final home = platform.environment['HOME'] ?? '~';
```

Comment:
~ is a shell concept, btw.  So Process won't expand it for you.

Reply:
yup I adjusted this, good call 👍